tdrs-backend/tdpservice/parsers/schema_defs/ssp/m6.py

```diff
@@ -339,7 +338,6 @@
 s3 = RowSchema(
     record_type="M6",
     document=SSP_M6DataSubmissionDocument(),
-    quiet_preparser_errors=True,
```

Comment:
@elipe17 same question

Reply:
Looks like this got clobbered in a merge somehow.